internal/oscap/main.go

```diff
@@ -0,0 +1,45 @@
+package oscap
```

Comment:
The file shouldn't be called `main.go` unless it has a `main()` function.  Typically the main (or only) file in a package is named after the package, so `oscap.go`.

Reply:
Ah cool thanks! I was wondering about that `main.go` didn't seem right, good to know about the convention.